### ISSUE TITLE
Calendar: demo need to show how just the calendar can be used

### ISSUE THREAD

**Issue by ThomasBurleson** [angular/material#4561]
We should create a demo to show how the Dialog, Calendar, and Card can be used to create something like:

![image](https://cloud.githubusercontent.com/assets/4432213/9680703/020fae30-52f9-11e5-8e5d-97b7ea43c600.png)

**Comment by jelbourn** [comment]
The main thing holding back `<md-calendar>` from being a standalone component is making sure the a11y is all correct; there's some interaction that only works when it's part of the datepicker.

**Comment by Deadly0** [comment]
Can I use the calendar how it shown on the picture above now? Or it will be available only from 0.12+?

**Comment by JobaDiniz** [comment]
I couldn't find `md-calendar` in the [docs](https://material.angularjs.org/0.11.2/#/). Where is it?

And how can I create a *theme* for [ui-calendar](http://angular-ui.github.io/ui-calendar/) plugin? It uses [fullcalendar](http://fullcalendar.io/), however, I cannot use `md-button` for example, because they render buttons with `button` tags.

**Comment by jelbourn** [comment]
@JobaDiniz The point of this issue is that there's no docs page for `md-calendar`.

**Comment by gligoran** [comment]
Any progress on this? I really need a stand-alone datepicker. I'd be happy with a codepen that shows how to do it, even if it's hacky. I think I'll prove less hacky than modifying other angular datepickers to material design.

**Comment by Deadly0** [comment]
@gligoran you can use this code to display calendar
```
<md-calendar ng-model="currentDate"></md-calendar>
```
but no docs for it.
Available params:
md-min-date
md-max-date
md-date-filter
it works the same way as in datepicker.

**Comment by gligoran** [comment]
@Deadly0 I tried that, but all it displays is the top row of day names.

It might need more work, I'm not aware of, so here's a pen: http://codepen.io/gligoran/pen/PZJxvm

**Comment by clavier-souris** [comment]
@gligoran `ng-cloak` on your main `div` is causing scroller container to be resized to `width: 0px`
I had the same exact problem.

A question of mine : is it possible to change the first day of week in a week row ?

**Comment by Deadly0** [comment]
@clavier-sourism yes, i use monday as the first day of a week

```
app.config(function($mdDateLocaleProvider)  {
	$mdDateLocaleProvider.firstDayOfWeek = 1;
});
```

**Comment by clavier-souris** [comment]
@Deadly0 Many thanks. Did not realize it was simply linked to locale configuration.

**Comment by gligoran** [comment]
@clavier-souris: Thanks.

Sadly, it looks like `mdMinDate`, `mdMaxDate`, and `mdDateFilter` don't update when the variables they're bound to, change. It makes sense, since when it's shown from the date picker, these won't change, but in standalone, they can.

**Comment by ThomasBurleson** [comment]
This issue is closed as part of our deprecation effort.
    For details, see our post [Spring Cleaning 2016](https://groups.google.com/forum/#!topic/ngMaterial/3Ip9nd68bfk).

**Comment by siefkenj** [comment]
It would be nice if this issue were re-opened.

**Comment by Splaktar** [comment]
Fixed in 1.1.10: https://material.angularjs.org/1.1.10/demo/datepicker
